src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
+
+- Added antiAffinity (configuration)[/docs/self-hosted/administration/configuration/#userpodaffinity] for PODs with label `dev.okteto.com/affinity`
+- Add external resource icons
+- Build credential helpers
+- Remove load time on GitHub redeploy dialog
+
+### Bug Fix
+
+- When injecting host alises to go through internal network to buildkit, registry and api, don't duplicate entries for same IP. Instead, pass the list of hostnames for the IP
+- Delete button is now disabled in the personal namespace
+- Restore k3s support
```

Comment:
- Since we never officially supported this, I think we should move this to features as "Support for k3s" . Do we support an specific version? @pchico83 are we ready to officially support this?

Reply:
@pchico83 @jpf-okteto @rberrelleza shoudl this be removed from release notes then?